src/palette.c

```diff
@@ -149,47 +149,43 @@ void ReadPlttIntoBuffers(void)
 bool8 BeginNormalPaletteFade(u32 selectedPalettes, s8 delay, u8 startY, u8 targetY, u16 blendColor)
 {
     u8 temp;
-    register u32 _blendColor asm("r8") = blendColor;
 
-    if (gPaletteFade.active)
+    if (gPaletteFade.active == TRUE)
```

Comment:
?????

Reply:
if (a == true) is NOT the same as if (a). they generate different code.